exercises/practice/list-ops/.docs/instructions.append.md

```diff
@@ -0,0 +1,101 @@
+# placeholder
+
+## New AWK concepts
+
+There are several interesting new concepts we're seeing in this exercise.
+
+### Include files
+
+This is the first exercise we've seen where the solution we're writing is
```

Comment:
In general, I try to avoid "we" or "you" in tech docs. Talk about the exercise and not the author.